packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -259,6 +264,13 @@ function leaveMeeting(meetingId) {
     });
 }
 
+function toggleMeetingInfo() {
+  const changeState = !webex.meetings.config.experimental.enableUnifiedMeetings;
```

Comment:
If you do this, the sample app page will always start off with the wrong API, instead of the default one. For instance, right now the default it's disabled, meaning "use v1 of meeting info" but when the app page gets loaded, it will use "v2 meeting info" even though it's not the default API. 
The same thing will happen when we enable v2 by default, it will put user on the wrong version

Reply:
Hi @lalli-flores 

This method is called only upon clicking the button `webex.meetings._toggleMeetingInfo()` and isn't called on load. 

In the samples app, it's enabled by default on webex.init. Kindly check [here](https://github.com/webex/webex-js-sdk/blob/b7ff9b8699564ea1dabfffa213479c414a003cec/packages/node_modules/samples/browser-plugin-meetings/app.js#L66). Therefore, it would take users to the right API as per the init. 

Kindly let me know if I'm on the same page as you are.

Reply:
@mkesavan13, you can try it yourself by going to a temporary link generated by this PR:
https://pr-2259.d3m3l2kee0btzx.amplifyapp.com/browser-plugin-meetings/

Reply:
@lalli-flores - I just checked. I think I understood your concern. Although by default the SDK has disabled unified meetings, the sample app enabled it by default during init.

Please check here -> [packages/node_modules/samples/browser-plugin-meetings/app.js#L55](https://github.com/webex/webex-js-sdk/blob/b7ff9b8699564ea1dabfffa213479c414a003cec/packages/node_modules/samples/browser-plugin-meetings/app.js#L55)

That's why, by default we have V2 init. The method I've written toggles between enable and disable based on what state it is currently in. Would you want me to change the init to disableUnifiedMeetings?

Reply:
@mkesavan13, thanks for pointing it out, I didn't realize that the flag was enabled for the app

Reply:
Ok @lalli-flores 

Will do that.

Reply:
https://github.com/webex/webex-js-sdk/pull/2270

Here's the PR for disabling Unified meetings by default in sample app @lalli-flores